core/ts/MaciState.ts

```diff
@@ -60,16 +64,13 @@ export class MaciState implements IMaciState {
    * @returns The index of the newly deployed poll.
    */
   public deployPoll(
-    duration: number,
```

Comment:
Why do we no longer need duration? Infer it from `pollEndTimestamp`?

Reply:
Seems like given the `Poll` contract accepts a `duration`, it makes sense to have?

Reply:
I removed it cause we never used it anywhere, we only ever used the end timestamp